src/index.js

```diff
@@ -1 +1,46 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+// import Image from './nasa2.png';
+
+const getLink = 'https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY&start_date=2018-07-15&end_date=2018-07-20';
+const main = document.getElementById('addToScreen');
+// const logo = document.querySelector('.logo');
+
+// function showImage() {
+//   const myImage = new Image();
+//   myImage.src = Image;
+//   return myImage;
+// }
+
+// logo.appendChild(showImage());
+
```

Comment:
[OPTIONAL] Kindly, remove all commented out code so that our code looks clean according to the best practices.